frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditCodeList/EditCodeList.tsx

```diff
@@ -31,6 +38,34 @@ export function EditCodeList<T extends SelectionComponentType>({
     });
   };
 
+  const handleInvalidFileName = (file?: FormData, fileName?: string) => {
+    const fileNameWithoutExtension = removeExtension(fileName);
+    if (isFileNameDuplicate(optionListIds, fileNameWithoutExtension)) {
+      toast.error(t('ux_editor.modal_properties_code_list_upload_duplicate_error'));
+    }
+  };
+
+  const handleUpload = (file: FormData) => {
+    uploadOptionList(file, {
+      onSuccess: () => {
+        for (const entry of file.entries()) {
+          if (entry[1] instanceof File) {
+            handleOptionsIdChange(removeExtension(entry[1].name));
+            toast.success(t('ux_editor.modal_properties_code_list_upload_success'));
+          }
+        }
+      },
+    });
+  };
+
+  const onValidateFileName = (fileName: string) => {
```

Comment:
Prefikset `on` bruker vi når en funksjon blir _utløst av_ en hendelse. Denne funksjonen blir ikke uløst av valideringen - den _utfører_ valideringen. Nå ser jeg at ved å kalle denne kun `validateFileName`, vil navnet kollidere med navnet på funksjonen den kaller videre, men vi kan for eksempel gi den et navn som `runValidation` eller noe i den dur.